include/render/fx_renderer/fx_renderer.h

```diff
@@ -141,20 +145,28 @@ struct fx_renderer {
 		PFNGLGETINTEGER64VEXTPROC glGetInteger64vEXT;
 	} procs;
 
-	struct {
-		struct quad_shader quad;
-		struct tex_shader tex_rgba;
-		struct tex_shader tex_rgbx;
-		struct tex_shader tex_ext;
-		struct box_shadow_shader box_shadow;
-		struct stencil_mask_shader stencil_mask;
-	} shaders;
+	struct shaders shaders;
```

Comment:
I think we actually want to keep this initialization as is here, as the renderer is the only place we use the `shaders` struct

Reply:
So you'd like to also move the shader.c impl into the renderer as well?

My reason for doing this is to make the fx_renderer.c file a bit shorter, making it easier for the eyes :)

Reply:
nope, just the `shaders` struct. I don't think it needs to exist outside of the renderer as it is only referenced there. I do think that the specific shader structs (`quad_shader`, `tex_shader` etc) are great to have in a separate file though as you've done since we do have functions that set them up that can live in a separate `shaders.c` file

Reply:
`link_shaders` could then be moved back to fx_renderer.c